modules/ec2-nat-instances/main.tf

```diff
@@ -69,10 +72,11 @@ module "ec2-nat" {
   iam_profiles       = ["${var.iam_profiles}"]
   security_group_ids = ["${var.security_group_ids}"]
   subnet_ids         = ["${var.public_subnet_ids}"]
+
   # let AWS choose IPs for these instances
```

Comment:
In the next section, removing the one extra space makes sense, but I find `fmt`'s annoying in the editor (interferes with multi-file editing).